workflow-templates/DataMiner-CICD-Automation-DeployOnly.yml

```diff
@@ -7,8 +7,9 @@ name: DataMiner DeployOnly Automation
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
-    branches: []
     tags:
```

Comment:
I think branches: [] was left in there to work together with:
if: github.ref_type == 'branch'

should we not keep that functionality? It could be useful for staging setups and testing.